lib/src/tabs/tab-set.component.ts

```diff
@@ -10,52 +10,36 @@ export function throwErrorForMissingRouterLink(tabsWithoutRouterLink: TabCompone
 }
 
 @Component({
-    template: `
-        <div class="hc-{{direction}}-tab-container">
-            <div class="hc-tab-bar-{{direction}}">
-                <div *ngFor="let tab of tabs">
-                    <a *ngIf="routerEnabled" class="hc-tab-{{direction}}"
-                       [routerLink]="tab.routerLink"
-                       routerLinkActive="active"
-                       (click)="setActive(tab)">
-                          {{tab.tabTitle}}
-                    </a>
-                    <a *ngIf="!routerEnabled" class="hc-tab-{{direction}}"
-                       [class.active]="tab.active"
-                       (click)="setActive(tab)">
-                          {{tab.tabTitle}}
-                    </a>
-
-                </div>
-            </div>
-            <div class="hc-tab-content-{{direction}}">
-                 <router-outlet *ngIf="routerEnabled"></router-outlet>
-                 <ng-content *ngIf="!routerEnabled"></ng-content>
-             </div>
-        </div>
-    `,
     selector: `hc-tab-set`,
+    templateUrl: './tab-set.component.html',
     styleUrls: ['../sass/_tabs.scss']
 })
 export class TabSetComponent implements AfterContentInit {
-    @ContentChildren(TabComponent) tabs: QueryList<TabComponent>;
-    @Input() direction: TabDirection = 'vertical';
     routerEnabled: boolean = false;
 
-    constructor(private router: Router, private route: ActivatedRoute) {}
+    @ContentChildren(TabComponent)
+    tabs: QueryList<TabComponent>;
+
+    @Input() direction: TabDirection = 'vertical';
+
+    constructor(private router: Router,
+                private route: ActivatedRoute) {
+    }
+
+    ngAfterContentInit(): void {
+        this.tabs.changes.subscribe(_ => {
```

Comment:
It seems to be convention to use `() => ` instead of `_ => `

Reply:
I was working with more functional libraries and so this convention slipped into this code. However, if you have a function that requires that you define an input but you don't care about using that input it's standard to use an underscore signifying that you don't care about that value.

The reason it works for subscribe to use empty parenthesis is because the next argument is defined to be optional. This won't work in all situations.